CHANGELOG.md

```diff
@@ -10,13 +10,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Adds support for token based authentication to support Kubernetes Authenticator
 
-## [2.0.0](https://github.com/cyberark/conjur-api-java/releases/tag/v2.0.0) - 2018-07-12
+## [2.0.0] - 2018-07-12
 ### Added
 - License updated to Apache v2 - [PR #8](https://github.com/cyberark/conjur-api-java/pull/8)
+
 ### Changed
 - Authn tokens now use the new Conjur 5 format - [PR #21](https://github.com/cyberark/conjur-api-java/pull/21)
-- Configuration change. When using environment variables, use `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY` now
-    instead of `CONJUR_CREDENTIALS` - https://github.com/cyberark/conjur-api-java/commit/60344308fc48cb5380c626e612b91e1e720c03fb
+- Configuration change. When using environment variables, use `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY` now instead of `CONJUR_CREDENTIALS` - https://github.com/cyberark/conjur-api-java/commit/60344308fc48cb5380c626e612b91e1e720c03fb
 
 ## [1.4.0] - 2015-05-15
```

Comment:
I don't see releases 1.4.0, 1.3.0, nor 1.2.0 on the releases page:  https://github.com/cyberark/conjur-api-java/releases, and I don't see `git tag` entry for these in the repo. Maybe these were deleted?  So probaby should delete these from the changelog.